packages/ckeditor5-coremedia-studio-integration/src/content/BlocklistService.ts

```diff
@@ -0,0 +1,56 @@
+import { Observable } from "rxjs";
+
+/**
+ * Service, used to manage a list of blocklisted words within CKEditor.
+ * Blocklisted words are words that should not appear in the editor content
+ * and should be taken care of by an editor.
+ * They are therefore highlighted within the text.
+ *
+ * This service interface allows to add or remove words and observe the whole list.
+ * It is used in the `ckeditor5-blocklist` plugin.
+ *
+ * The service has to be registered globally by `serviceAgent` and may then
```

Comment:
> The service has to be registered globally by `serviceAgent`

I think it has to be registered at the service agent (or on). But by seems wrong because the service agent is not responsible for the registering services.